extendr-api/tests/dataframe_tests.rs

```diff
@@ -0,0 +1,43 @@
+#[test]
+fn dataframe_tests() {
+    use extendr_api::prelude::*;
+    use extendr_api::wrapper::Dataframe;
+    test! {
+        let x = [(1_i32, 1.0_f64), (2, 2.0), (3, 3.0)];
+
+        let df : Dataframe<(i32, f64)> = Dataframe::try_from_values(x).unwrap();
```

Comment:
To be honest, this is a little bit confusing. We use `DataFrame` but with rust types. Shouldn't it be `(Rint, Rfloat)` ? 
To me, any tabular data type should either be fully rust (`Table<(i32, f64)>` backed by som Rust type, `Vec<>` or `HashMap<>`) or fully R (`DataFrame<(Rint, Rfloat)>` backed by `SEXP`). If we mix these two approaches, we will get all sorts of confusing results.

Reply:
I've dropped the Tuple derive as it is hard to implement and confuses.